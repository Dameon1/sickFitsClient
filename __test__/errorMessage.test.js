import DisplayError from '../components/ErrorMessage';
import {  mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import wait from 'waait';

describe('<DisplayError/>', () => {
  it('displays the error message given', async () => {
    const wrapper = mount(
       <DisplayError error={{error:'simple error message'}} />
       );
    await wait();
    wrapper.update(); 
    expect(toJSON(wrapper)).toMatchSnapshot();
  })
});