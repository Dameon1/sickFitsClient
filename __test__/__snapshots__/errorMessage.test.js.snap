// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayError/> displays the error message given 1`] = `
<DisplayError
  error={
    Object {
      "error": "simple error message",
    }
  }
/>
`;
